README.md

```diff
@@ -34,25 +40,7 @@ estraverse.traverse(ast, {
 });
 ```
 
-### Document
-
-Generated JSDoc is [here](http://estools.github.io/escope/).
-
-### Demos and Tools
-
-Demonstration is [here](http://mazurov.github.io/escope-demo/) by [Sasha Mazurov](https://github.com/mazurov) (twitter: [@mazurov](http://twitter.com/mazurov)). [issue](https://github.com/estools/escope/issues/14)
-
-![Demo](https://f.cloud.github.com/assets/75759/462920/7aa6dd40-b4f5-11e2-9f07-9f4e8d0415f9.gif)
-
-
-And there are tools constructed on Escope.
-
-- [Esmangle](https://github.com/estools/esmangle) is a minifier / mangler / optimizer.
-- [Eslevels](https://github.com/mazurov/eslevels) is a scope levels analyzer and [SublimeText plugin for scope context coloring](https://github.com/mazurov/sublime-levels) is constructed on it.
-- [Esgoggles](https://github.com/keeyipchan/esgoggles) is JavaScript code browser.
-
-
-### License
+### Original Escope License
```

Comment:
We don't actually need this, as the original license is in the LICENSE file. 

Also, we need a section on contributing, build commands, and current license. You can copy those from the TypeScript parser readme.